lib/optimizely/version.rb

```diff
@@ -17,5 +17,5 @@
 #
 module Optimizely
   CLIENT_ENGINE = 'ruby-sdk'
-  VERSION = '3.10.1'
+  VERSION = '4.0.0'
```

Comment:
We only update version when we are making a PR for release prep. you should leave the SDK version as is for this PR